appcues/src/main/java/com/appcues/analytics/AutoPropertyDecorator.kt

```diff
@@ -55,7 +54,9 @@ internal class AutoPropertyDecorator(
             "userId" to storage.userId,
             "_isAnonymous" to storage.isAnonymous,
             "_localId" to storage.deviceId,
-            UPDATED_AT_PROPERTY to Date(),
+            "_updatedAt" to Date(),
+            // Last Seen At will deprecate _updatedAt which can't be entirely removed today since it's used for targeting
+            "_lastSeenAt" to Date(),
```

Comment:
nit, but might be best to have a shared `now` variable that is used in both spots, so they are guaranteed to have identical value, like https://github.com/appcues/appcues-ios-sdk/pull/496/files#diff-ded0c79ea6f55775f64fabb6e0cb83e0d69f4cd0a0912523386714135db2450dR69